src/elements/hx-beacon/_shadow.html

```diff
@@ -0,0 +1,3 @@
+<div id="hxDismiss">
+  <hx-beacon-circle></hx-beacon-circle>
```

Comment:
This is a good effort.

Since `<hx-beacon-circle>` is not a custom element used/provided in `HelixUI`, please change to a element specific to the `<hx-beacon>` Shadow DOM.  Please try using a `<span id="hxBeaconPulse>` or `<div id="hxBeaconPulse>`.

Reply:
Changes have been done, please review